test/erlfmt_markdown_SUITE.erl

```diff
@@ -136,29 +136,37 @@ check_markdown(Content) ->
 split_code_into_maps(_Text, {text, Formatted, Unformatted}) ->
     {code, Formatted, Unformatted};
 split_code_into_maps(Text, {code, Formatted, Unformatted}) ->
-    [FirstLine, Code] = string:split(Text, "\n"),
+    [FirstLine, Code0] = string:split(Text, "\n"),
+    Code = strip_good_bad(Code0),
     Spec = string:split(FirstLine, " ", all),
     case Spec of
-        ["erl" ++ _, ToFormat | Key] ->
-            case ToFormat of
-                "formatted" ->
-                    {text, maps:put(Key, Code, Formatted), Unformatted};
-                "unformatted" ->
-                    {text, Formatted, maps:put(Key, Code, Unformatted)}
-            end;
+        ["erl" ++ _, "formatted" | Key] ->
+            {text, maps:put(Key, Code, Formatted), Unformatted};
+        ["erl" ++ _, "unformatted" | Key] ->
+            {text, Formatted, maps:put(Key, Code, Unformatted)};
         _ ->
             {text, Formatted, Unformatted}
     end.
 
+strip_good_bad("%% Good\n" ++ Rest) -> Rest;
```

Comment:
Maybe we can just use headings, instead of comments?

Reply:
I tried that, but it looked very "spacious" when rendered

Reply:
Smaller headings?

Reply:
Or even just text

Reply:
This is how it renders now
![Screenshot 2020-07-24 at 15 58 14](https://user-images.githubusercontent.com/477062/88405011-911c6f80-cdc6-11ea-95d4-d5294cdde76c.png)
This is how it renders with raw text
![Screenshot 2020-07-24 at 15 57 57](https://user-images.githubusercontent.com/477062/88405001-8d88e880-cdc6-11ea-8c37-85bf5c4d4bc3.png)
I think it makes the annotation way to prominent compared to the code itself

Reply:
Just wondering if maybe we can use the tick and cross from the formatting decisions instead of the words, if there is a way to align it better.

Reply:
But I think lets merge what we have I will try to see if I can come up with something.  I just really dislike the magic or removing specific words, but I guess it is fine for now and then I really started disliking the word `Bad`.